### ISSUE TITLE
Maximum call stack size exceeded RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by vatechie** [myspace-nu/jquery.fancyTable#8]
Hi,
I am getting the maximum call stack exceeded error in chrome when the number of records exceed 10k limit.
Jquery doesn't do the pagination of html content and throws the below error in console.
jQuery.Deferred exception: Maximum call stack size exceeded RangeError: Maximum call stack size exceeded
    at Function.se [as find] (http://localhost:4200/scripts.js:2:6350)
    at k.fn.init.find (http://localhost:4200/scripts.js:2:24471)
    at HTMLTableElement.<anonymous> (http://localhost:4200/scripts.js:126:22)
    at Function.each (http://localhost:4200/scripts.js:2:2777)
    at k.fn.init.each (http://localhost:4200/scripts.js:2:1419)
    at k.fn.init.$.fn.fancyTable (http://localhost:4200/scripts.js:119:8)
    at HTMLDocument.<anonymous> (http://localhost:4200/main.js:1688:36)
    at e (http://localhost:4200/scripts.js:2:29453)
    at t (http://localhost:4200/scripts.js:2:29755)
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (http://localhost:4200/polyfills.js:2785:31) undefined

While it works find in firefox.
Can you please provide any idea what would be causing this issue.

**Comment by myspace-nu** [comment]
I don't think this is an issue or bug related to fancyTable itself. I tested to raise the number of rows generated by the load simulation in the example file to 15k, and that does not recreate the issue you're describing (tested in current Chrome version 81). However, the page is sluggishly slow to generate all rows. Some quick google searches suggested that you might have too many events registered on the page. fancyTable only registers click events for the number of pagination buttons used where the default setting will only result in 9.

To sum things up. If you have > 10k rows you might want to consider using another approach and filter them out on the server side.

**Comment by vatechie** [comment]
Hi @myspace-nu , 
Thanks for your input.
Fancytable works fine in firefox browser but not in chrome in case of more than 10k records.
Actually I am getting html content from the API so can't trim the data at the API end.
Is it possible to restrict Facnytable to paginate only 10k records and ignore the rest.

Regards,
Vishal

**Comment by myspace-nu** [comment]
I'm still not sure that adding a "fix" to a problem that is not related to fancyTable itself is the right way to go. As I mentioned, I have used 15k rows in Chrome without breaking functionallity. My guess is that there are several things that contributes to the problem and not just the number of rows in the table. Things like what's actually contained in the rows. Are there other huge elements and events that slow down the DOM?

If trimming the number of rows is a solution in your case, it can easily be achieved by something like:
```
var maxRows=10000;
var rowCount=$("table#mytable tr").length;
$("table#mytable").find("tr:nth-last-child(-n+"+(rowCount-maxRows)+")").remove();
```

Before applying fancyTable.
